stock_picking_portal/models/res_config_settings.py

```
@@ -13,25 +13,37 @@ class ResConfigSettings(models.TransientModel):
     )
 
     def set_values(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `set_values`

(ANN201)